packages/codemirror/src/editor.ts

```diff
@@ -1316,6 +1329,23 @@ namespace Private {
     return a.line === b.line && a.ch === b.ch;
   }
 
+  /**
+   * Get the list of active gutters
+   *
+   * @param config Editor configuration
+   */
+  function getActiveGutters(config: CodeMirrorEditor.IConfig): string[] {
+    // The order of the classes will be the gutters order
+    let classToSwitch = {
+      'CodeMirror-linenumbers': 'lineNumbers',
+      'CodeMirror-foldgutter': 'codeFolding'
+    };
+    return Object.keys(classToSwitch).filter(
+      // @ts-ignore
+      gutter => config[classToSwitch[gutter]]
```

Comment:
Do you know how I can get ride of the `ts-ignore`? I don't know how to get a `ts-lint` compliant code here.